server/local/studentdashboard-DEV-permissive.yml

```diff
@@ -0,0 +1,67 @@
+### This yaml format file specifies configuration values that may change from
```

Comment:
What's the difference between this file and the previous studentdashboard-DEV-permissive-file.yml file? Do we really need two?